CMakeLists.txt

```diff
@@ -88,12 +95,10 @@ set(ENABLE_PROFILING "OFF" CACHE BOOL
 
 SET(TYPE_ARCHITECTURE "native" CACHE STRING "Choose the type of architecture to compile to in release mode. Default is to let the compiler decide")
 
-
 message(STATUS "Build type is: ${CMAKE_BUILD_TYPE}")
 
 ########## COMPILATION FLAGS ##########
 add_compile_options(-Wall -Wextra)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Weffc++")
```

Comment:
Accidentally removed it, added it back now. Is it fine now?

Reply:
Okay I know remember the issue. Google benchmark does not satisfy -Weffc++ therefore I removed it globally and added it as target to the tests. Now I put the global -Weffc++ flag back and only removed the flag for the compilation of the benckmarks
```
  # removes the Weffc++ from the compilation of the benchmarks
  string(REPLACE "-Weffc++" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
  all files related to google benchmark ....
  # puts the Weffc++ flag back to the compilation process
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Weffc++")
```

This is related to, where I did not remember the issue with the Weffc.
https://github.com/cosmo-epfl/librascal/pull/113#discussion_r332509978

Reply:
I managed that there is no compiler error due to Weffc++ but there are a lot of compiler warnings due to Weffc++. Is it okay If I remove the global set Weffc++ again and add it to the target of the tests, examples, ... everything except benchmarks?

Reply:
I'll have a look at the warnings, I tend to agree, since it is relying on an external library. But I'm not 100% following. With `-Werr` warnings become errors, right?